airflow/providers/redis/log/redis_task_handler.py

```diff
@@ -75,6 +75,8 @@ def _read(
         ti: TaskInstance,
         try_number: int,
         metadata: dict[str, Any] | None = None,
+        offset: int = 0,
+        limit: int = 100,
```

Comment:
No need to change the redis_task_handler

Reply:
Same thing with above with es. Is this fine?